build(app-functions): Remove pre-build script for app functions master branch

rsdmike opened the pull request:
As we move to making go 1.12 a requirement, tests are failing due to `shell/install_custom_golang.sh` installing 1.11.5. In talking to @lranjbar, removing this `pre_build_script`, should resolve the failure for new pull requests coming into `master` for app functions as it will install go 1.12.

Fixes #419 

Signed-off-by: Mike Johanson <michael.johanson@intel.com>

lenny-goodell reviewed (dismissed):
LGTM

lranjbar reviewed (dismissed):
:+1:

eb-oss reviewed (changes requested):
Could you please update the commit message with some body text explaining why this change is being made?

rsdmike committed:
Remove pre-build script for app functions master branch

As we move to making go 1.12 a requirement, tests are failing due to shell/install_custom_golang.sh installing 1.11.5. In talking to @lranjbar, removing this pre_build_script, should resolve the failure for new pull requests coming into master for app functions as it will install go 1.12.

Fixes #419

Signed-off-by: Mike Johanson <michael.johanson@intel.com>

jamesrgregg reviewed (approved):
LGTM - @rsdmike thanks for updating the commit message
